TSC-Charter.md

```diff
@@ -17,60 +17,47 @@ In the case of the Node.js project, it is delegated to the Node.js
 Technical Steering Committee (“TSC”). OpenJS Foundation’s business
 leadership is the Board of Directors (the “Board”).
 
-This Technical Steering Committee Charter reflects a carefully
-constructed balanced role for the TSC and the CPC in the governance of
-the OpenJS Foundation. The charter amendment process is for the TSC to
-propose changes using simple majority of the full TSC, the proposed
-changes being subject to review and approval by the CPC. The CPC may
-additionally make amendments to the TSC charter at any time, though the
-CPC will not interfere with day-to-day discussions, votes or meetings
-of the TSC.
+The charter amendment process is for the voting members of the TSC to propose
+for review and approval by the CPC. The CPC may additionally make amendments to
+the TSC charter at any time, though the CPC will not interfere with day-to-day
+discussions, votes or meetings of the TSC.
 
 ## Section 3. Establishment of the TSC
 
 TSC memberships are not time-limited. There is no maximum size of the TSC.
-The size is expected to vary in order to ensure adequate coverage of important
-areas of expertise, balanced with the ability to make decisions efficiently.
-The TSC must have at least four members.
+The TSC must have at least four voting members.
```

Comment:
I think you might need to define voting and non-voting members somewhere? Right now there's some amount left unsaid.

Presumably, the new rules imply that one has to be added to the TSC as a voting member and can only transition to non-voting membership by failing to meet the participation criteria?

Similarly, the "removal" aspect becomes a bit less clear. The wording right now still requires resignation/removal of TSC voting members due to employee affiliation quotas but wouldn't being moved to a regular member be enough?

Reply:
Yes, room for improvement/clarity there for sure. (I did this very quickly because I suspected that if I didn't do it in a 15-minute sprint, I probably wouldn't come back to it for a long time.)

Reply:
I've added a paragraph at the start of section 3 that explains what regular members and voting members are. I'm going to optimistically mark this as resolved, but feel free to unresolve it or leave an additional comment if you think more should be done.